cmd/juju/controller/killstatus.go

```diff
@@ -91,6 +91,10 @@ func newData(api destroyControllerAPI, controllerModelUUID string) (ctrData, []m
 		return ctrData{}, nil, errors.Trace(err)
 	}
 
+	// This is needed since return parameter may have an Error
+	// property in the later api versions, > 3.
+	newerAPI := api.BestAPIVersion() > 3
```

Comment:
is this really needed? model.Error will always be nil if api version <= 3, so just do a blind check on model.Error?

Reply:
Removed.